lmnet/executor/profile_model.py

```diff
@@ -136,7 +159,12 @@ def _profile_flops(graph, res, scopes):
         elif scope in float_res_dict:
             new_res.append([1, scope, "-", "-", "-", flops])
 
-    return new_res
+    node_flops_dict = {
+        'total_flops': float_res_dict["total"],
+        'children': [{"name": k, "flops": float_res_dict[k]} for k in float_res_dict.keys() if k != "total"]
```

Comment:
`[{"name": k, "flops": v for k, v in dict.items()]` ?

Reply:
Changed 👍